### PR TITLE
Change flash messages

### PR DESCRIPTION
**Pull request by munckymagik**
Updates the bold headings for the flash messages:
pasted image at 2015_10_06 11_27 am

... and also updates the individual flash messages for added/saved records.

### CODE REVIEW

**Comment by Tr4pSt3R** [comment]
Awesome :+1:

File: spec/controllers/selfservice/advisers_controller/create_spec.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ module SelfService
         end
 
         it 'adds a success flash message' do
-          expect(flash[:notice]).to eq('Saved successfully')
+          expect(flash[:notice]).to eq(I18n.t('self_service.adviser_edit.saved'))
```
Comment (rnalexander) [inline spec/controllers/selfservice/advisers_controller/create_spec.rb]: Total stickler point here.  Any reason why you're using the 'I18n.t' notation here but the shorter 't' notation in the _alerts.erb file?

Comment (munckymagik) [inline spec/controllers/selfservice/advisers_controller/create_spec.rb]: IIRC the shorter `t` version is a helper in the views and thus doesn't need a prefix. Elsewhere we need to call the method of the same name on `I18n` itself. Certainly, I just took off the prefix in this text and re-ran the test, it failed, couldn't resolve `t`, `NoMethodError`.

Comment (munckymagik) [inline spec/controllers/selfservice/advisers_controller/create_spec.rb]: And I should add, is that ok? It seemed like that was "the way things are done".

**Comment by rnalexander** [comment]
All good!  :+1: :shipit:

**Comment by munckymagik** [comment]
Thanks

**Pull request merged**